tests/test_routes.py

```diff
@@ -71,3 +72,35 @@ def test_search(client):
         ).json
 
         assert len(response_json) == len(rows)
+
+
+@pytest.mark.parametrize("sub_criteria_response_keys", ["id", "name", "score_submitted","themes"])
```

Comment:
Bit of a nitpick, but this parametrize seems unnecessary since we just have a single input

Reply:
I think my plural naming misleads here, but each time we are only testing for a single response key, so we need to parametrize to test for each key. Good spot ill change the name to reflect this.